### ISSUE TITLE
How to verify problem 8?

### ISSUE THREAD

**Issue by mgranados** [workshopper/learnyoureact#103]
I see the HTML outputted is the right answer but by running learnyoureact verify I get the following error:  
![screen shot 2016-04-29 at 1 19 13 pm](https://cloud.githubusercontent.com/assets/1660748/14925431/0e4cb65a-0e0d-11e6-8553-4c9dce984b69.png)

**Comment by theatlasroom** [comment]
I'm getting this same issue too

**Comment by konato** [comment]
Same issue here to

**Comment by data-doge** [comment]
+1

**Comment by koustuvsinha** [comment]
same issue!

**Comment by FukKwang** [comment]
Same problem ! can anyone help ?

**Comment by Sticksword** [comment]
+1

**Comment by jsdavis** [comment]
Same issue. Even copy/pasted the solution files and got failed verification

**Comment by a-metta** [comment]
is still still happening? what are the solutions that failed verification? I've just completed the tutorial and have been able to verify properly.

**Comment by Sticksword** [comment]
I copy pasted the same things as the solution repo and still had verification errors. I don't know why

**Comment by a-metta** [comment]
this is my package.json:

{
  "name": "learnyoureact",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "babel": "^5.8.23",
    "body-parser": "^1.15.2",
    "express": "^4.14.0",
    "express-react-views": "^0.9.0",
    "react": "^0.14.8",
    "react-dom": "^0.14.8"
  }
}

are the dependencies the same?

**Comment by Sticksword** [comment]
Mine's this:

```
{
  "name": "learnyoureact",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "babel": "^5.8.23",
    "babel-preset-es2015": "^6.9.0",
    "babel-preset-react": "^6.11.1",
    "babelify": "^7.3.0",
    "body-parser": "^1.15.2",
    "browserify": "^13.0.1",
    "express": "^4.14.0",
    "express-react-views": "^0.9.0",
    "react": "^15.2.1",
    "react-dom": "^15.2.1"
  }
}
```

It would seem the only difference is `react` and `react-dom`. Do you think that's the cause?

**Comment by rossb** [comment]
+1. I'm getting the same error during verification as in the screenshot above. Knowing little about React, it does look like something caused by latest React version producing different HTML than the workshop expects.

**Comment by a-metta** [comment]
yes I would try to install that version that i have posted considering how react has been going through some major changes

**Comment by rossb** [comment]
Just a quick note to say I resolved this problem by deleting my node_modules folder, replacing the dependencies list in my local package.json with the one [from the repo itself](https://github.com/kohei-takata/learnyoureact/blob/master/package.json) then running `npm install`. The problem verifies OK after doing that.

**Comment by xaca** [comment]
Funciona, borrando la carpeta de node_modules luego npm install como dice @rossb

**Comment by BlessYAHU** [comment]
setting your package json to 0.14.0 for react and react-dom packages (effectively, removing the ~), also works.  Maybe there should be a notice put on the read me about this?

**Comment by jedau** [comment]
@rossb 's suggestion solves it. It was literally right in front of us lol! Thanks for the reality check, mate.

**Comment by johnykov** [comment]
I have a problem with this one, I managed to verify solution, code is being copied from solutions dir. But I see proper html file only after first page load and then when I reload or I try to navigate to /bundle.js I see this error in console:

```
events.js:160
      throw er; // Unhandled 'error' event
      ^

TypeError: Cannot read property 'error' of undefined (While processing preset: "/Users/jankowalski/projects/nodeschool/learnyoureact/node_modules/babel-preset-es2015/index.js") while parsing file: /Users/jankowalski/projects/nodeschool/learnyoureact/app.js
```
I use @rossb's dependencies

**Comment by krischamp** [comment]
@hanskoff 
I got the same pb as your's ... but following rossb's solution I got it right ... 
just verify after the first time and it should be ok (or if it's still out, go to next CSS2 and verify back)

**Comment by nharrisanalyst** [comment]
im getting the same error as @hanskoff

when I verify it works but when I run  node program.js 
I get the error and can not load the page at localhost:3000

**Comment by kaz080** [comment]
@krischamp @nharrisanalyst 
Maybe #111 will be of some help.

**Comment by dkonieczek** [comment]
Changing the following worked for me.
```
require('babel/register')({
    ignore: false
});
```
replace with:
```
require('babel/register');
```

**Comment by mjohnson518** [comment]
Hi,

I'm having an issue with learnyoureact Isomorphic exercise. I continue to get an error:

var body = DOM.body;
^
Cannot read property 'body' of undefined

Can you assist? Thanks

**Comment by maxrumsey** [comment]
@mjohnson518 Have you tried this: https://github.com/workshopper/learnyoureact/issues/103#issuecomment-239604351?
